clean-abap/CleanABAP.md

```diff
@@ -551,6 +551,21 @@ Try to build things in a clean, object-oriented way.
 If something is too slow, make a performance measurement.
 Only then should you take a fact-based decision to discard selected rules.
 
+Some further thoughts taken in part from Chapter 2 of Martin Fowler's book "Refactoring":
```

Comment:
Let's link the book here: https://martinfowler.com/books/refactoring.html

Reply:
Citation style would be:
`Some further thoughts, taken in part from Chapter 2 of [Martin Fowler's book _Refactoring_](https://martinfowler.com/books/refactoring.html):`

Reply:
OK, done.